applications/crossbar/src/modules/cb_transactions.erl

```diff
@@ -79,7 +79,9 @@ allowed_methods(?CURRENT_BALANCE) ->
     [?HTTP_GET];
 allowed_methods(?DEBIT) ->
     [?HTTP_DELETE];
-allowed_methods(_) ->
+allowed_methods(?MONTHLY) ->
+    [?HTTP_GET];
```

Comment:
omagad thank you so much for this stuff! Reading crossbar modules was pain because of blackholes like these.

Thank you for taking the time to do this!